microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml

```diff
@@ -134,6 +134,18 @@
          <attribute name="failureRatio" range="[0,1]" /> <!-- 0 <= x <= 1 -->
          <attribute name="successThreshold" range="1" /> <!-- x >=1 -->         
       </annotationValidator>
+
+    <annotationValidator annotation="org.eclipse.microprofile.faulttolerance.Bulkhead"
+                   source="microprofile-faulttolerance" >
+         <attribute range="0" />
```

Comment:
I would still remove this attribute as it causes the entire validation to fail for Bulkhead.